config/dev.exs

```diff
@@ -62,7 +63,7 @@ config :ret, Ret.Repo,
   username: "postgres",
   password: "postgres",
   database: "ret_dev",
-  hostname: "localhost",
+  hostname: "db",
```

Comment:
Not sure if there is a better way to configure the database host name specifically for docker-compose.

Reply:
hmm yeah i am not sure. on the fence which one makes more sense.